app/models/broadcast_message.py

```diff
@@ -181,7 +181,9 @@ def approved_by(self):
 
     @cached_property
     def cancelled_by(self):
-        return User.from_id(self.cancelled_by_id)
+        if not self.cancelled_by_id:
+            return "API integration"
```

Comment:
```suggestion
            return "an API call"
```

This is the terminology we use [elsewhere](https://github.com/alphagov/notifications-admin/blob/2635c80ca4659483cd01dd72d9da577ab5b653e8/app/templates/views/broadcast/view-message.html#L200). I prefer having the text as part of the method here so it's all in one place 👍.

Reply:
Good spot, I have amended this.